tests/regression/router-sanitizer.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+source tests/regression/common.sh
+
+check_env
+setup_sanitizer_opts
+
+# Get command line arguments
+# Current arguments parsed are <sleep_time> <remaining_args>
+get_cli_args $@
+
+# Install prerequisites
+make
+pip install --user -r tests/regression/requirements.txt
+redis-server &
+
+# Iterate over all available build options
+for (( i = 0; i < ${#SAN_OPTIONS[@]}; i++ )); do
+    option=${SAN_OPTIONS[${i}]}
+
+    trap 'TA_INIT=1' USR1
+    bazel run accelerator ${option} -c dbg -- --ta_port=${TA_PORT} &
+    TA=$!
+    
+    TA_INIT=0
+    while [ "$TA_INIT" -ne 1 ]; do
+        if ps -p $TA > /dev/null; then
+            echo "waiting tangle-accelerator initialization"
+            sleep 1
+            continue
+        else
+            # pid does not exist
+            break
+        fi
+    done
+    # FIXME: Send SIGINT before bash file get the pid of tangle-accelerator will 
+    # cause bash script can be terminated in the next line
```

Comment:
Please separate the comment with two or more sentences.
It is hard to understand.